core/src/main/java/pl/agh/edu/management/hotel/HotelScenariosManager.java

```diff
@@ -6,27 +6,35 @@
 
 import pl.agh.edu.enums.HotelType;
 import pl.agh.edu.enums.HotelVisitPurpose;
+import pl.agh.edu.json.data.AttractivenessConstantsData;
 import pl.agh.edu.json.data_loader.JSONHotelScenariosDataLoader;
 import pl.agh.edu.model.HotelPopularityFunction;
 import pl.agh.edu.model.time.Time;
 
 public class HotelScenariosManager {
 	private EnumMap<HotelVisitPurpose, Double> hotelVisitPurposeProbabilities;
 	private Map<MonthDay, Double> seasonalMultiplier;
+
+	public AttractivenessConstantsData getAttractivenessConstants() {
```

Comment:
Jakby mozna było przenieść to trochę niżej niż atrybuty( zakładam że generowałeś z intellij i tam wrzuciło)